src/continuous-validation.cc

```diff
@@ -47,7 +47,11 @@ namespace hpp {
     {
       interval.first = -std::numeric_limits <value_type>::infinity ();
       interval.second = std::numeric_limits <value_type>::infinity ();
-      pinocchio::DeviceSync robot (robot_);
+
+      robot_->currentConfiguration (config);
+      robot_->computeFramesForwardKinematics ();
+      robot_->updateGeometryPlacements ();
```

Comment:
This will break thread safety. Why do you need this ?

Reply:
Without those lines, the robot configuration is not updated in my case. I am using a derived class of ContinuousValidation::Dichotomy with a derived class of hpp::manipulation::Device.

Reply:
Does your derived device store some data ?

In hpp::pinocchio::Device, all the data are stored in hpp::pinocchio::DeviceData.
Note that your derived collision validation must used the DeviceSync object to get the updated datas.

Reply:
Indeed, that is actually my problem, my derived collision validation is not using the DeviceSync object. I will remove those lines and fix my code accordingly.

Reply:
Ok, thanks. I can provide guidelines if you need some.